packages/web/src/components/TeeResults.vue

```diff
@@ -45,21 +45,38 @@
       @click="updateDetailResult(prog.id)">
       <div class="fr-card__body">
         <div class="fr-card__content">
-          <h2 class="fr-card__title tee-program-resume">
+          <!-- TITLE -->
+          <div class="fr-card__start fr-mb-2v">
+            <p 
+              class="tee-program-title">
+              {{ prog.titre }}
+            </p>
+          </div>
+          <!-- CONTENT -->
+          <h2 
+            class="fr-card__title tee-program-resume fr-mb-3v"
+            style="font-size: 1.7rem; line-height: 2rem;">
             {{ prog.promesse }} 
           </h2>
+          <!-- DEBUG -->
           <p
             v-if="debug"
             class="vue-debug fr-card__desc">
             <br> choices.publicPath : <code>{{ choices.publicPath }}</code>
             <br> prog.cover : <code>{{ prog.illustration }}</code>
             <!-- {{ `${choices.publicPath}${randomImage()}` }} -->
           </p>
-          <!-- TITLE -->
-          <div class="fr-card__start">
+          <!-- END -->
+          <div class="fr-card__end">
             <p 
-              class="fr-mb-3v tee-program-title">
-              {{ prog.titre }}
+              class="fr-mb-0"
+              style="color: #000091;">
+              <span 
+                class="fr-icon-money-euro-circle-line" 
+                aria-hidden="true">
+              </span>
+              {{ choices.t(getCostInfosPrefix(prog)) }} :
```

Comment:
question: Why not use the function `choices.t` directly on `getCostInfosPrefix` function ?

Reply:
good catch